.travis.yml

```diff
@@ -54,7 +54,21 @@ matrix:
             - silversearcher-ag
             - expect-dev  # unbuffer
             - coreutils
+            # dependencies for licensecheck
+            - cpanminus
             - libyaml-tiny-perl
+            - libio-socket-ssl-perl
```

Comment:
Should be added in the INSTALL file for contributors?

Reply:
This is solely needed for travis though - it's not intended as something which is run by developers (same with the /usr/bin/perl change below)